# Service to Get Transformation of Mocap w.r.t Robot Frame.

---
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
int64 ret
string msg
